code/modules/events/pirates.dm

```diff
@@ -146,15 +146,9 @@
 		new /obj/effect/temp_visual/emp(get_turf(S))
 
 /obj/machinery/shuttle_scrambler/proc/dump_loot(mob/user)
-	if(credits_stored < 200)
-		to_chat(user,"<span class='notice'>Not enough credits to retrieve.</span>")
-		return
-	while(credits_stored >= 200)
-		new /obj/item/stack/spacecash/c200(drop_location())
-		credits_stored -= 200
+	new /obj/item/holochip(drop_location(), credits_stored)
```

Comment:
As a result of this being its own thing, it now won't stack and will only pull out 200 credits. Undocumented change. I highly advise you just make this a subtype of space cash codewise.

Reply:
unless i misread it, it will pull all the loot into a single chip, not 200.